huaweicloud/resource_huaweicloud_fgs_function_v2.go

```diff
@@ -125,6 +118,77 @@ func resourceFgsFunctionV2() *schema.Resource {
 					}
 				},
 			},
+			"depend_list": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"initializer_handler": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"initializer_timeout": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"vpc_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				RequiredWith: []string{"subnet_id"},
+			},
+			"subnet_id": {
```

Comment:
In order to keep consistent with other resources, the parameter should be `network_id`

Reply:
fixed